.github/workflows/coq-action.yml

```diff
@@ -26,23 +26,6 @@ jobs:
         with:
           opam_file: ${{ matrix.opam_name }}.opam
           coq_version: ${{ matrix.coq_version }}
-          custom_script: |
-            # This mostly follows https://github.com/coq-community/docker-coq-action/tree/v1.0.0
-            startGroup Opam config
-              opam repo add -y prerelease file://$WORKDIR/opam-prerelease
```

Comment:
@MSoegtropIMC removing this line didn't break the CI. If sometime you want to add it back, you can configure [`before_install`](https://github.com/coq-community/docker-coq-action#before_install) instead of the entire `custom_script`.